test/test_cuda.py

```diff
@@ -2896,6 +2896,209 @@ def test_max_large_axis(self):
     def test_to_numpy(self):
         self.assertRaises(TypeError, lambda: torch.empty(1, device="cuda").numpy())
 
+    @unittest.skipIf((not TEST_CUDA) or
+                     TEST_WITH_ROCM or
+                     int(torch.version.cuda.split(".")[0]) < 11, "CUDA >= 11.0 required for graphs")
+    def test_graph_capture_simple(self):
+        s1 = torch.cuda.Stream()
+
+        with torch.cuda.stream(s1):
+            a = torch.zeros((1000,), device="cuda")
+            a += 1
+            g = torch.cuda.Graph()
```

Comment:
This API name don't look very private to me! 🤣

Reply:
:shushing_face: 
Alright, I can not import it as Graph for now, so usage (for experimenters) would be
```
g = torch._C._CudaGraphBase()
g.capture_begin/capture_end/replay()
```

Reply:
Just make it `torch.cuda._Graph` for now, rename when we have a better idea what we want to do with it.